### PR TITLE
LLM: use pip to install bigdl-llm

### PR DESCRIPTION
**Pull request by WeiguangHan**
@Oscilloscope98 Please have a look. Looking forward to your comments. Thanks!

### CODE REVIEW

**Comment by Oscilloscope98** [comment]
image
These lines are also needed to be commented out

File: .github/workflows/llm_performance_tests.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ jobs:
         # pip install transformers_stream_generator for model internlm-chat-7b-8k
         # pip install tiktoken for model Qwen-7B-Chat-10-12
         run: |
+          python -m pip install --pre --upgrade bigdl-llm[xpu_2.1] -f https://developer.intel.com/ipex-whl-stable-xpu
           python -m pip install --upgrade pip
           python -m pip install --upgrade wheel
           python -m pip install --upgrade omegaconf
```
Comment (Oscilloscope98) [inline .github/workflows/llm_performance_tests.yml:57]: We may install bigdl-llm after pip and wheel :)

**Comment by WeiguangHan** [comment]
> image These lines are also needed to be commented out

Sure. Have commented them.

File: .github/workflows/llm_performance_tests.yml (hunk 1 above)
Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml:57]: sure:)

File: .github/workflows/llm_performance_tests.yml [hunk 2]
```diff
@@ -51,6 +51,7 @@ jobs:
         # pip install transformers_stream_generator for model internlm-chat-7b-8k
         # pip install tiktoken for model Qwen-7B-Chat-10-12
         run: |
+          python -m pip install --pre --upgrade bigdl-llm[xpu_2.1] -f https://developer.intel.com/ipex-whl-stable-xpu
```
Comment (hkvision) [inline .github/workflows/llm_performance_tests.yml:54]: jist [xpu] which is equivalent to xpu_2.1 by default

Comment (hkvision) [reply .github/workflows/llm_performance_tests.yml:54]: Also can we move the 66-69 lines?

Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml]: sure

File: .github/workflows/llm_performance_tests.yml [hunk 3]
```diff
@@ -53,19 +53,20 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install --upgrade wheel
+          python -m pip install --pre --upgrade bigdl-llm -f https://developer.intel.com/ipex-whl-stable-xpu
```
[the remark this answers is not in the document]
Comment (WeiguangHan) [reply .github/workflows/llm_performance_tests.yml:56]: ok.

**Comment by hkvision** [comment]
The changes LGTM.

**Pull request closed without merging**